includes/templates/omise-payment-form.php

```diff
@@ -1,30 +1,32 @@
 <div id="omise_cc_form">
-<?php 
-	$showExistingCards = $viewData["user_logged_in"] && isset($viewData["existingCards"]->data) && sizeof($viewData["existingCards"]->data) > 0;
-	if($showExistingCards){
-?>
-<p class="form-row form-row-wide">
-	Select card : <br/>
-		<?php 
-			foreach($viewData["existingCards"]->data as $card){
-				echo "<input type='radio' name='card_id' value='{$card->id}' />Card ends with {$card->last_digits}<br/>";
+	<?php
+		$showExistingCards = $viewData['user_logged_in'] && isset( $viewData['existingCards']->data ) && sizeof( $viewData['existingCards']->data ) > 0;
+		if ( $showExistingCards ) {
```

Comment:
@nimid for `<?php condition block ?>` inside html template, can you change `if { ... }` to `if : ... endif;` instead.
i.e.
```html
<?php if ( $a ) : ?>
    <li>Hi</li>
<?php endif; ?>
```
thanks

ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style